app/scripts/modules/azure/serverGroup/configure/serverGroupAdvancedSettingsSelector.directive.html

```diff
@@ -49,13 +49,13 @@
   </div>
 </div>
 <div class="form-group">
-  <div class="col-md-5 sm-label-right"><b>Ramdisk Id (optional)</b></div>
```

Comment:
Yes very good point - that explains why I couldn't find this UI :) Removed these files. We actually use the advancedSettings pages under 'wizard/advancedSettings/'